components/oc-date-time-now/public/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+(function (Bullet) {
+  var div = document.getElementById('date-time-now');
+  setInterval(function () {
+    div.innerHTML = Date.now();
+  }, 1000);
+
+  var counter = 0;
+  var setCounter = function setCounter() {
+    document.getElementById('spn-counter').innerHTML = counter;
+  };
+
+  setCounter();
+
+  document.getElementById('btn-increase').addEventListener('click', function () {
+    counter += 1;
+    setCounter();
+    if (Bullet) {
+      Bullet.trigger('oc-date-time-now:increase', counter);
```

Comment:
Even better as it will be one less dependency we have to use. I only need to make sure it is available for the React app as well. I'll try it later.